project/src/js/ui/tournament/tournamentCtrl.js

```diff
@@ -0,0 +1,51 @@
+import socket from '../../socket';
+import * as utils from '../../utils';
+import * as xhr from './tournamentXhr';
+import socketHandler from './socketHandler';
+import helper from '../helper';
+import m from 'mithril';
+
+const tournament = m.prop([]);
```

Comment:
`tounament` here should be a private field in function `controller` instead of defined globally.